__tests__/endpoints/articles.test.js

```diff
@@ -118,4 +118,69 @@ describe("/api/articles/:article_id", () => {
             })
         });
     });
+});
+
+describe("/api/articles/:article_id/comments", () => {
+    describe("GET", () => {
+        describe("if valid article_id", () => {
+            test("200: responds with an array of comments for given article_id", () => {
+                return request(app)
+                    .get("/api/articles/1/comments")
+                    .expect(200)
+                    .then((response) => {
+                        const comments = response.body.comments;
+                        expect(comments.length).toBe(11);
+
+                        comments.forEach((comment) => {
+                            expect(comment).toMatchObject({
+                                comment_id: expect.any(Number),
+                                votes: expect.any(Number),
+                                created_at: expect.any(String),
+                                author: expect.any(String),
+                                body: expect.any(String),
+                                article_id: expect.any(Number),
+                            });
+                        });
+                    });
+            });
+
+            test("200: comments by default are sorted by date (ASC)", () => {
+                return request(app)
+                    .get("/api/articles/1/comments")
+                    .expect(200)
+                    .then((response) => {
+                        expect(response.body.comments).toBeSortedBy('created_at', {descending: false});
+                    });
+            });
+
+            test("404: responds with error message if no article exists for given article_id", () => {
+                return request(app)
+                    .get("/api/articles/99/comments")
+                    .expect(404)
+                    .then((response) => {
+                        expect(response.body.msg).toBe("No article/comments found");
```

Comment:
Is this error message for no article found or no comments found?

Because technically if they request on a valid article but there are no comments found... that's fine right? Simply no one's commented on the article yet which is fair.
For a non-existent article it's a completely different story